yarn.lock

```diff
@@ -2783,6 +2783,18 @@
     jest-util "^29.6.1"
     slash "^3.0.0"
 
+"@jest/console@^29.7.0":
```

Comment:
This and every other dependency installed in this PR should be consolidated if possible with overlapping semver ranges. E.g. `@jest/console@^29.6.1` and `@jest/console@^29.7.0` can both use v29.7.0.

The way to do this is, for each dependency you wish to consolidate:
1. Manually edit the `yarn.lock` file and delete all the entries for the dependency you wish to try and consolidate
2. Run `yarn cache clean <name-of-package>`
3. Run `yarn kbn bootstrap`

Reply:
I'll revert these changes as the upgrade wasn't necessary for this PR